common/Configuring_CloudForms.adoc

```diff
@@ -0,0 +1,155 @@
+[[Configuring-cloudforms]]
+== Configuring {product-title}
+
+Although the {product-title} appliance comes configured to be integrated immediately into your environment, you can make some changes to its configuration.
+
+[NOTE]
+======
+The {product-title} appliance is intended to have minimal configuration options.
+======
+
+=== Changing Configuration Settings
```

Comment:
Per the article title comment above, headers within the doc should start at L2 (two '='):

    == Changing Configuration Settings